packages/types/src/index.ts

```diff
@@ -77,19 +88,30 @@ export interface Confirm {
  * Action Types
  */
 
-export type KnownAction = UsersSelect | StaticSelect | ConversationsSelect | ChannelsSelect | ExternalSelect |
-  Button | Overflow | Datepicker;
+ // Selects and Multiselects are available in different surface areas so I've seperated them here
+export type KnownSelect = UsersSelect | StaticSelect | ConversationsSelect | ChannelsSelect | ExternalSelect;
```

Comment:
It doesn't seem like there's any reason to prefix these two types with `Known`. The type no longer gets used in unions with `Action`, so all the action types are "known" unless they are `Action`.